src/AiidaDFTK.jl

```diff
@@ -101,7 +102,36 @@ Run a DFTK calculation from a json input file.
 Output is by default written to `stdout` and `stderr`.
 The list of generated output files is returned.
 """
-function run_json(filename::AbstractString; extra_output_files=String[])
+function run_json(filename::AbstractString; extra_output_files=String[], min_version=typemin(VersionNumber), max_version=typemax(VersionNumber))
+    # We write a few key messages to an error file that we can parse from
+    # the AiiDA plugin to detect common failure modes.
+    errorfile = "errors.log"
```

Comment:
maybe use process id as well to make the file name unique ?

Reply:
That would make it difficult (impossible?) to retrieve through AiiDA since we have to give filenames ahead of time.

Reply:
Ok, true, but the list of files is dumped somewhere, no ?

Reply:
It looks like AiiDA supports glob patterns with `*` so we could make it work. However I don't really see the point. After all, we use fixed file names for the other outputs as well.

Reply:
Ok.